jenkins/ci.suse.de/openstack-mkcloud.yaml

```diff
@@ -166,6 +174,7 @@
            - raw
            - rbd
            - netapp
+           - nfs
```

Comment:
If this is set then the additional step to install the nfsserver needs to be run as well, right?